src/Scratch/Install-Sitecore12Databases.ps1

```diff
@@ -0,0 +1,48 @@
+<#
```

Comment:
Eventually Install-SitecoreDevInstanceDatabases would make sense. Not sure if you guys plan to use this wrapper for any other situations than local dev? (and if it is only used on dev, do we need it?)